### DISCUSSION TITLE
[Icon request] Tabler icons set

### DISCUSSION THREAD

**Discussion by yoyurec** [ryanoasis/nerd-fonts#935] [Ideas]
I'd like to request include to Nerd fonts an icon pack **Tabler**
https://github.com/tabler/tabler-icons
![image](https://user-images.githubusercontent.com/137919/191404876-93d76b14-8144-4f93-a3c1-431b55f3caa4.png)

> Over 2650 pixel-perfect icons for web design
> Free and open source icons designed to make your website or app attractive, visually consistent and simply beautiful.

**Comment by rubiin** [comment]
+1

**Comment by Finii** [comment]
This has Issue #1814 for further conversation. Comments in the Discussions section often went unnoticed.
